content/02.main-text.md

```diff
@@ -152,6 +152,7 @@ Pandoc supports conversion between additional formats — such as LaTeX, AsciiDo
 [Journal Article Tag Suite](https://jats.nlm.nih.gov/) (JATS) is a standard XML format for scholarly articles that is used by publishers, archives, and text miners [@url:http://www.niso.org/standards/z39-96-2015/; @doi:10.6087/kcse.2014.1.99; @doi:10.1080/00987913.2012.10765464].
 Pandoc's JATS support provides an avenue to integrate Manubot with the larger JATS ecosystem.
 For now, the primary Manubot output is HTML intended to be viewed in a web browser.
+Accordingly, manuscripts natively support javascript and can thus include any web-based interactive visualization.
```

Comment:
Is JavaScript the formal capitalization?

Do we want to name any specific interactive visualization libraries here?  I'm not sure a reader will read `web-based interactive visualization` and think of graphics like what plotly, D3, Bokeh, etc. offer.